plugin-server/src/main/services/http-server.ts

```diff
@@ -0,0 +1,36 @@
+import { createServer, IncomingMessage, Server, ServerResponse } from 'http'
+
+import { healthcheck } from '../../healthcheck'
+import { status } from '../../utils/status'
+import { stalenessCheck } from '../../utils/utils'
+import { Hub, PluginsServerConfig } from './../../types'
+
+const HTTP_SERVER_PORT = 5000
+
+export function createHttpServer(hub: Hub | undefined, serverConfig: PluginsServerConfig): Server {
+    const server = createServer(async (req: IncomingMessage, res: ServerResponse) => {
+        if (req.url === '/health' && req.method === 'GET') {
+            const status = await healthcheck()
+            const ok = status ? !stalenessCheck(hub, serverConfig.HEALTHCHECK_MAX_STALE_SECONDS).isServerStale : false
```

Comment:
I might be wrong / missing something but I don't think using a staleness check to validate the health of a service can work. 

Sample scenario: 
1. `t0`: the plugin-server finish to ingest an event
1. `t0 + 1ms`: the connection between the plugin-server and a 3rd party service breaks (e.g. Kafka)
1. the `/health` endpoint will return 200 till `t0 + HEALTHCHECK_MAX_STALE_SECONDS` despite the service is unhealthy

Reply:
What I replied to you on Slack:

> I think the one aspect that tries to address is the issue we just had with Ne****a where the plugin server was healthy but events were never being picked up from Kafka. That was the takeaway I had from standup but could be wrong. The other check does cover things being up and running though, although we could extend it too